sampleinput.yaml

```diff
@@ -0,0 +1,14 @@
+creds:
+  github:
+    token: my_token
+  gitlab:
+    token: my_token
+    host: my_hostname
+git:
+   - github:
+     - user_name/repo_name
+     - user_name
+   - gitlab:
```

Comment:
For github and pagure, you only need read access to list pull requests from any repository(regardless of you being author or contributor or just third party), so you could just list them down and there wouldn't be additional possibility of instances if I understood the meaning of instance correctly.

for gitlab, I am bit skeptical since I feel you might not be able to use same tokens for different host_server. Will do more testing.